components/ui/Header.tsx

```diff
@@ -19,16 +19,29 @@ export const Header: React.FC<HeaderProps> = ({goBack, title, testID}) => {
             marginVertical: 16,
           }}>
           <TouchableOpacity onPress={goBack} {...testIDProps('goBack')}>
-            <Icon
-              name="arrow-left"
-              type="material-community"
-              onPress={goBack}
-              containerStyle={{
-                ...Theme.Styles.backArrowContainer,
-                marginLeft: 10,
-              }}
-              color={Theme.Colors.Icon}
-            />
+            {I18nManager.isRTL ? (
```

Comment:
Did you check it's impact on other screen like QRConsent, MyBindedVCs?

Reply:
verified on the both the QRConsent and MyBindedVCs screens working as eexpected